iotdevice/mux.go

```diff
@@ -235,6 +235,13 @@ func (m *methodMux) remove(method string) {
 func (m *methodMux) Dispatch(method string, b []byte) (int, []byte, error) {
 	m.mu.RLock()
 	f, ok := m.m[method]
+	wildcard := false
+	if !ok {
+		f, ok = m.m["#"]
```

Comment:
I think an empty string method name would be better choice for the fallback handler, since mux itself doesn't support mqtt wildcards.

Reply:
I would prefer a magic string because in most cases empty strings have other significant meaning.  There is already a check for empty string on registration that would need to be removed, but I'm flexible.

Reply:
Sorry for the late response, I'd prefer an empty string as the default wildcard, having this would give me idea that mqtt topic syntax is fully supported here.